dgl/src/Window.cpp

```diff
@@ -19,6 +19,10 @@
 
 #include "../Base.hpp"
 
+#if defined(DISTRHO_OS_MAC)
+#import <Cocoa/Cocoa.h>
```

Comment:
This part is not right, I messed up a previous change in pugl_osx.m where Cocoa is imported too late.
The fix should be done in the pugl files.

Reply:
In the pugl files it's too late, because the header sequence `OpenGL.hpp` → `ImageBase.hpp` → `Geometry.hpp` has been included at this point, and the types `Point` and `Size` will conflict.

Reply:
You're right, I forgot this..
I added something in develop branch that should fix this.
The DGL namespace is not "exported" when building dgl, only for the plugins. So the conflict should not happen anymore.

Reply:
Rebased on latest `develop` branch